test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ex
+
+test_image=api-test
+docker build -t $test_image -f Dockerfile.test .
+
+function finish {
+	docker rm -f $pg_cid
+	docker rm -f $server_cid
+}
+trap finish EXIT
+
+possum_tag=push-image_170626_0.1.0
```

Comment:
This will need to be updated from `push-image_170626` to `master` after https://github.com/conjurinc/possum/pull/74 gets merged.